src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -260,6 +260,26 @@ public abstract partial class TransportContext
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
 }
+namespace System.Net.Cache
+{
+    public enum RequestCacheLevel
+    {
+        Default = 0,
+        BypassCache = 1,
+        CacheOnly = 2,
+        CacheIfAvailable = 3,
+        Revalidate = 4,
+        Reload = 5,
+        NoCacheNoStore = 6
+    }
+    public class RequestCachePolicy
+    {
+        public RequestCachePolicy() { }
+        public RequestCachePolicy(System.Net.Cache.RequestCacheLevel level) { }
+        public System.Net.Cache.RequestCacheLevel Level { get { throw null; } }
```

Comment:
Yes, they've all been changed across the repo.  It's simpler in code and results in less IL (especially if there are out parameters).